src/assets/i18n/cy.json

```diff
@@ -192,7 +290,9 @@
       "manualHandling": "Codi a chario",
       "slalomFigure8": "Slalom / Ffigwr 8",
       "slowControl": "Rheolydd araf",
-      "uTurnExercise": "Ymarfer tro 'U'"
+      "uTurnExercise": "Ymarfer tro 'U'",
```

Comment:
these are also in cat a mod 1 and are in @faggcs pull request unsure of this will affect whoever merges first